main.go

```diff
@@ -555,99 +564,116 @@ func processLines(header []string, numChars int, config *Config, queue chan [][]
 					arrowBuilder.WriteRow(arrowRow)
 				}
 
-				if multiallelic {
-					output.WriteString(parse.NotTrTv)
-				} else {
-					output.WriteString(parse.GetTrTv(string(refs[i]), alts[i]))
-				}
+				if !config.noOut {
```

Comment:
from here to the end of the function has not been changed, just indented due to the new condition (!config.noOut)